packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -236,3 +237,153 @@ def get_task_url(dcos_api_session, task_name, stream=False):
     return '/system/v1/agent/{}/logs/v1/{}/framework/{}/executor/{}/container/{}'.format(slave_id, endpoint_type,
                                                                                          framework_id, executor_id,
                                                                                          container_id)
+
+
+def validate_journald_cursor(c: str, cursor_regexp=None):
+    if not cursor_regexp:
+        cursor_regexp = b'^id: s=[a-f0-9]+;i=[a-f0-9]+;b=[a-f0-9]+;'
```

Comment:
This regex will definitely need a comment. 
Also, why is a byte object - prefixed with `b`? That is confusing.  The string should be sufficient.

```
$ python -q
>>> import re
>>> re.compile("^id:").match("id: matches")
<_sre.SRE_Match object; span=(0, 3), match='id:'>
```